src/xdocs/config_annotation.xml

```diff
@@ -810,7 +810,17 @@ public static final int COUNTER = 10; // violation
 
       <subsection name="Examples" id="PackageAnnotation_Examples">
         <p> To configure the check:</p>
-        <source> &lt;module name=&quot;PackageAnnotation&quot;/&gt;
+        <source> &lt;module name=&quot;PackageAnnotation&quot;/&gt; </source>
+        <p>Examples of validating files package-info.java and MyClass.java:</p>
+        <p>package-info.java     //ok</p>
```

Comment:
Again, on your screenshot you have a single whitespace but here you use few of them.

Reply:
Kept single white space.